GameData/SPEngine/Parts/Gamma_proc.cfg

```diff
@@ -60,28 +56,11 @@
 		name = ModuleSPEngine
 		familyLetter = X
 	}
-	@MODULE[ModuleGimbal]
-	{
-		%gimbalRangeXP = 8
-		%gimbalRangeXN = 8
-		%gimbalRangeYP = 0
-		%gimbalRangeYN = 0
-		%useGimbalResponseSpeed = true
-		%gimbalResponseSpeed = 12
-	}
```

Comment:
Why are you getting rid of my gimbal settings?  They're deliberate, and (afaik) not the same as the RD0110 we're cloning.